tests/unit/pubsublite/internal/wire/subscriber_impl_test.py

```diff
@@ -464,11 +465,13 @@ async def test_handle_reset_signal(
         )
 
         # Send messages to the subscriber.
-        await read_result_queue.put(as_response([message_1, message_2]))
+        await read_result_queue.put(as_response([message_1]))
+        await read_result_queue.put(as_response([message_2]))
 
         # Read one message.
         await read_called_queue.get()
-        assert (await subscriber.read()) == message_1
+        batch1 = await subscriber.read()
+        assert batch1[0].SerializeToString() == message_1.SerializeToString()
```

Comment:
I'm not sure I understand why the comparison is done on serialized messages now.

Reply:
Python protos (real protos, not proto-plus) don't implement __eq__, so its an identity check otherwise.